resources/assets/sass/_components.scss

```diff
@@ -210,7 +210,6 @@ body.flexbox-support #entity-selector-wrap .popup-body .form-group {
 
 .image-manager-sidebar {
   width: 300px;
-  margin-left: 1px;
```

Comment:
Not sure why this was there. Adds a bit of gap which I think doesn't look very good.